.yamllint.yml

```diff
@@ -0,0 +1,85 @@
+yaml-files:
```

Comment:
What's the reason we don't use yamllint's default configuration ?

Reply:
1. I find the default configuration a little to permissive for braces and bracket
2. I prefer the config to be explicite

Reply:
I'd rather have one less configuration file to worry about

> I prefer the config to be explicit

I disagree: it's better to have one way to do the formatting and don't think about it (see for instance black for Python or rust-fmt compared to the mad house that is C formatting)

> I find the default configuration a little to permissive for braces and bracket

See my previous point ;-)